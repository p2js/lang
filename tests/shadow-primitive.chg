mod u8 {}
fn f(x: u8) {}
fn main() {}
