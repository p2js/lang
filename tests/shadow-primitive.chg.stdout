expr @ 2:13 3:0 => ()
expr @ 3:11 4:1 => ()



define void @main() {
prologue:
br label %bb.0

bb.0:
ret void

}


